### PR TITLE
Scix 137 implement paper network

### PR DESCRIPTION
**Pull request by shinyichen**
https://user-images.githubusercontent.com/636361/187455154-c32aa057-e69f-437e-89bf-4cd26244bad8.mov

Screen Shot 2022-08-31 at 3 58 00 PMScreen Shot 2022-08-31 at 3 52 24 PM

### CODE REVIEW

**Review by thostetler — APPROVED** [review]
This looks pretty good to me! 
My only complaint would be that for some of the functions added to the components as "helpers" essentially -- I think it would be better to move them outside the component so their references can be reused in subsequent renders. 
I've been a fan recently of moving most of the logic into pure functions in a helpers file next to the component.

Anyway, this looks good - had a couple of small things I noticed.

👍

File: src/api/vis/models.ts [hunk 1]
```diff
@@ -1,6 +1,12 @@
-import { Bibcode } from '@api/search';
+import { IADSApiSearchParams } from '@api/search';
 import { IADSApiVisParams } from './types';
 
-export const getAuthorNetworkParams = (bibcodes: Bibcode[]): IADSApiVisParams => ({
-  bibcodes,
-});
+export const getAuthorNetworkParams = ({ q, sort, rows }: IADSApiSearchParams): IADSApiVisParams => {
+  const query = JSON.stringify({ q: [`${q}`], sort: [`${sort.join(',')}`], rows: [`${rows}`] });
```
[the remark this answers is not in the document]
Comment (shinyichen) [reply src/api/vis/models.ts:5]: Good point, I saw this as the request in BBB, but I just tested without the `[]` and still worked. Will fix this.

**Comment by thostetler** [comment]
One small thing I noticed (and this is also on BBB) that when you click on a paper it opens in a new tab and takes you away from the graph, I wonder if it would be difficult to render the abstract page in a slide out from the side (https://chakra-ui.com/docs/components/drawer) or similar to how Hollis does it: https://hollis.harvard.edu/primo-explore/search?query=any%2Ccontains%2Cnasa&tab=everything&search_scope=everything&vid=HVD2&offset=0

Could be pretty cool for seeing the abstract/other info quickly, could always open the full page then.
If we were to do it we should make a full component that takes in a bibcode and makes the calls to pull the info for rendering. 

Anyway just a thought!

**Comment by shinyichen** [comment]
I like this idea. I would like to try it out.

> One small thing I noticed (and this is also on BBB) that when you click on a paper it opens in a new tab and takes you away from the graph, I wonder if it would be difficult to render the abstract page in a slide out from the side (https://chakra-ui.com/docs/components/drawer) or similar to how Hollis does it: https://hollis.harvard.edu/primo-explore/search?query=any%2Ccontains%2Cnasa&tab=everything&search_scope=everything&vid=HVD2&offset=0
> 
> Could be pretty cool for seeing the abstract/other info quickly, could always open the full page then. If we were to do it we should make a full component that takes in a bibcode and makes the calls to pull the info for rendering.
> 
> Anyway just a thought!

**Pull request merged**